irohad/main/irohad.cpp

```diff
@@ -73,6 +73,11 @@ DEFINE_string(keypair_name, "", "Specify name of .pub and .priv files");
  */
 DEFINE_validator(keypair_name, &validate_keypair_name);
 
+/**
+ * Creating boolean flag to overwrite ledger data if existing
```

Comment:
"Creating"? With that words sounds strange. What do you want to say by that?

Reply:
I tried to use the same words that were used for other arguments in this same file like : 
 * `Creating input argument for the keypair files location.`
 * `Creating input argument for the genesis block file location.` 

Should instead use `Defining boolean flag ...` ?

Reply:
Ah, I see. I think `flag to overwrite` -> `flag for overwriting` sounds quite better